gazelle_cabal/lang.go

```diff
@@ -205,11 +206,35 @@ func (*gazelleCabalLang) Fix(c *config.Config, f *rule.File) {
 	ruleInfos := cabalToRuleInfos(cabalFiles)
 
 	for _, r := range f.Rules {
+		print(r)
+
 		if !r.ShouldKeep() &&
 			isHaskellRule(r.Kind()) &&
 			!hasRuleInfo(ruleInfos, r.Kind(), r.Name()) {
 			r.Delete()
 		}
+
+		if !r.ShouldKeep() &&
+			r.Kind() == "stack_snapshot" {
+			print("Case accessed")
+			var list []string
+			pack := r.Attr("packages")
+			switch expr := pack.(type) {
+			case *bzl.ListExpr:
+				for _, elem := range expr.List {
+					switch exprElem := elem.(type) {
+					case *bzl.StringExpr:
+						list = append(list, exprElem.Value)
+					default:
+						panic("Elements of packages should be string!")
+					}
+				}
+			default:
+				panic("Packages should be a list!")
+			}
+			print(list)
+			r.SetAttr("packages", listSortedStringKeys(list))
```

Comment:
Is reordering the strings necessary? If so, perhaps it should be mentioned in the README.